nvm.sh

```diff
@@ -2419,13 +2421,51 @@ nvm() {
             PROVIDED_REINSTALL_PACKAGES_FROM="$(nvm_echo "$1" | command cut -c 22-)"
             REINSTALL_PACKAGES_FROM="$(nvm_version "$PROVIDED_REINSTALL_PACKAGES_FROM")" ||:
           ;;
+          --skip-default-packages)
+            SKIP_DEFAULT_PACKAGES=true ||:
```

Comment:
`||:` shouldn't be required here, since you're not doing anything that could possibly have a nonzero exit code.